mappings/net/minecraft/server/gui/StatComponent.mapping

```diff
@@ -1,4 +1,5 @@
 CLASS none/lk net/minecraft/server/gui/StatComponent
+	FIELD a FORMAT Ljava/text/DecimalFormat;
```

Comment:
What FORMAT? Is it just a FORMAT? It's probably not just a FORMAT, I refrained from naming it before.